core/types.go

```diff
@@ -39,6 +39,7 @@ const (
 	DutyExit                DutyType = 4
 	DutyBuilderProposer     DutyType = 5
 	DutyBuilderRegistration DutyType = 6
+	DutySignature           DutyType = 7
```

Comment:
instead of breaking backwards compatibility in DKG and charon run, only break charon run, so like issue mentions: DutyRandao should be 7 and DutySignature should remain at 3.